docs/zkapps/how-zkapps-work.mdx

```diff
@@ -122,19 +125,19 @@ After a zkApp is deployed to a host (e.g. mycoolzkapp.com), users can interact w
-4. User clicks “submit to chain” in the zkApp UI and their wallet (such as a browser extension wallet) will prompt them to confirm sending the transaction. The wallet signs the transaction containing the proof and associated description of state to update and sends it to the Mina blockchain.
-5. When the Mina network receives this transaction, it will verify that the proof successfully passes the verifier method listed on the zkApp account. If the network accepts this transaction, this indicates that this proof and the requested state changes are valid and, as such, are allowed to update the zkApp’s state.
+3. The prover function in the zkApp generates a zero-knowledge proof locally using the data entered by the user. This data can be either private (which is never be seen by the blockchain) or public (which is stored either on chain or off chain), depending on what the developer specified, as needed for a given use case. Additionally, a list of state updates (called events) to be created by this transaction is generated and is associated with this proof. 
+4. User clicks “submit to chain” in the zkApp UI and their wallet (such as a browser extension wallet) prompts them to confirm sending the transaction. The wallet signs the transaction that contains the proof and associated description of state to update and sends it to the Mina blockchain.
+5. When the Mina network receives this transaction, it verifies that the proof successfully passes the verifier method listed on the zkApp account. If the network accepts this transaction, this indicates that this proof and the requested state changes are valid and, as such, are allowed to update the zkApp’s state.
 
-Because the user’s interaction occurs _locally_ within their web browser (using JavaScript on the client), the user’s privacy can be maintained.
+Because the user’s interaction occurs _locally_ within their web browser (using JavaScript on the client), the user’s privacy is maintained.
 
 ### How is state updated on chain
 
-You may be wondering how the state on a zkApp account gets updated on chain. We glossed over that detail earlier to keep things simple.
+Now you have learned enough to wonder how the state on a zkApp account gets updated on chain. 
 
-When the prover function runs in a web browser, the smart contract outputs a proof and some associated data that we call “events”. This is sent as part of the transaction when sending a transaction to a zkApp address. These events are a plaintext description (in JSON) describing how to update the state on a zkApp account.
+When the prover function runs in a web browser, the smart contract outputs a proof and some associated data called “events”. This is sent as part of the transaction when sending a transaction to a zkApp address. These events are a JSON plaintext description that describes how to update the state on a zkApp account.
```

Comment:
@barriebyron the naming here has changed. Can we update "events" to "account updates" in these two paragraphs please?

(Note: This is describing something different than "events" in the recently reviewed "actions & events" docs. It's describing the components of a zkApp transaction, which are one or more "account updates".

Reply:
good catch, please review carefully to ensure I get it right here